Access_Control_Policy.md

```diff
@@ -0,0 +1,68 @@
+# ACCESS CONTROL POLICY (DRAFT)
+
+## Classification Level
+Publicly available
+
+## Review Information
+
+__Mandatory Review Period__
+
+Yearly
+
+__Date of Last Review__
+
+N/A
+
+## Introduction
+
+An access control policy helps employees and external parties understand who is allowed to access what client and user data 
+and under what circumstances and conditions they can access that data.
+
+### Goal Statement
+
+At present, we are a small company (literally everyone in the company knows everyone else on a first name basis). The 
+unfortunate reality is that, no matter how much we may know, like, or trust an individual we know personally, they have the 
+capacity to take harmful actions. Beyond that, there is risk that an employee's identity (from a system perspective) may be 
+compromised allowing an attacker the same level of access as an employee. In general, we mitigate these kinds of risks by 
+collecting as little data as possible so that, even in the event of a breach, nothing of value would be lost. However, we do
+offer a service whose value is in its availability and providing too much access to everyone could result in a loss of that 
+availability to our clients. To that end we need some basic guidelines indicating who should have access to what, and how,
+when, and why that access should change.
+
+### Background Statement 
+
+We, at Fonticons, know our culture but it is entirely reasonable for others planning to use our technology to desire to 
```

Comment:
Fonticons, Inc. (?)

Reply:
While these are policies they are probably by-laws more than traditional, legally binding docs.  If you think Fonticons causes confusion I can give the full legal name.